app/controllers/application_controller.rb

```diff
@@ -1,5 +1,17 @@
 class ApplicationController < ActionController::Base
-  def self.current_user
-    User.first
+  # def self.current_user
+  #   User.first
+  # end
+  protect_from_forgery with: :exception
+
+  protected
+
+  def update_allowed_parameters
+    devise_parameter_sanitizer.permit(:sign_up) do |u|
+      u.permit(:name, :email, :password, :password_confirmation, :bio)
+    end
```

Comment:
- [ ] I was not able to signup for your application. When I entered the details required for sign up and clicked the signup button nothing happened, no messages were shown, I was not directed to the login page and asked to confirm my email in the terminal. Kindly ensure that Devise has been installed and set up using the [getting started link](https://github.com/heartcombo/devise).

Reply:
This issue is well resolved! ✔️